src/components/dialogs/create-case-dialog/create-case-dialog.tsx

```diff
@@ -162,23 +160,13 @@ const CreateCaseDialog: React.FunctionComponent<ICreateCaseDialogProps> = ({
         }
     };
 
-    const [caseFileAdornment, caseNameChecking]: NameCheckReturn = useNameCheck(
-        {
+    const [caseFileAdornment, caseNameChecking]: NameCheckReturn =
+        useNameCheck<IFormData>({
             field: CASE_NAME,
-            name: caseName,
+            value: getValues(CASE_NAME),
```

Comment:
Nope, please keep the previous `watch`, we'll change this later in the TS you took.